roads.mss

```diff
@@ -3224,10 +3236,10 @@ tertiary is rendered from z10 and is not included in osm_planet_roads. */
         text-spacing: 300;
         text-clip: false;
         text-placement: line;
-        text-min-distance: 18;
         text-face-name: @book-fonts;
         text-halo-radius: @standard-halo-radius;
         text-halo-fill: @standard-halo-fill;
+        text-repeat-distance: @text-repeat-distance;
```

Comment:
This code is not working properly in my tests - see my previous comment on `text-repeat-distance` values.